wsl-vpnkit-start.sh

```diff
@@ -66,11 +86,13 @@ if [ "${EUID:-"$(id -u)"}" -ne 0 ]; then
 fi
 
 relay &
-sleep 3
+while [ ! -S "${SOCKET_PATH}" ]; do
+  sleep 0.001
+done
 vpnkit &
-sleep 3
+sleep 3 # Is this needed?
 tap &
-sleep 3
+sleep 3 # Is this needed?
 ipconfig
```

Comment:
To be honest I copied this whole file from sakai135. The second sleep can probably be omitted if we remove the ipconfig. I don't see what value that adds